cmd/prometheus-config-reloader/main.go

```diff
@@ -145,11 +145,16 @@ func main() {
 			cancel()
 		})
 	}
-
+	f := func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(`{"status":"up"}`))
+		w.WriteHeader(http.StatusOK)
```

Comment:
```suggestion
		w.Write([]byte(`{"status":"up"}`))
```
Could you please test if `Write` before `WriteHeader` produces a "mutiple status code set" warning. I think it is sufficient to just use Write and not set the status explicitly.

Reply:
thinks for U review. `Write` before `WriteHeader` not find warning. `WriteHeader` just set data to http protocol header segment, `Write` set data to http protocol body segment.

Reply:
```
package main

import (
	"fmt"
	"log"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		w.Write([]byte("Hello, World!"))
		w.WriteHeader(http.StatusOK)
	})

	fmt.Println("listening ... curl http://localhost:8888/foobar ")
	log.Fatal(http.ListenAndServe(":8888", nil))
}
```
Does produce this warning ... 

```
listening ... curl http://localhost:8888/foobar
2023/03/28 15:36:19 http: superfluous response.WriteHeader call from main.main.func1 (write_header.go:12)
```

From the doc you can see that 

```
// Write writes the data to the connection as part of an HTTP reply.
	//
	// If WriteHeader has not yet been called, Write calls
	// WriteHeader(http.StatusOK) before writing the data.
```
see: https://pkg.go.dev/net/http#ResponseWriter

Reply:
Thinks for U review. Have Fixed.